extensions/logging-gelf/runtime/src/main/java/io/quarkus/logging/gelf/GelfConfig.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.logging.gelf;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "log.handler.gelf")
+public class GelfConfig {
+    /**
+     * Determine whether to enable the GELF logging handler
+     */
+    @ConfigItem
+    boolean enabled;
```

Comment:
This will be `false` by default; was that intentional?

Reply:
Yes, I try to use the same construct as in https://github.com/quarkusio/quarkus/blob/master/extensions/logging-json/runtime/src/main/java/io/quarkus/logging/json/runtime/JsonConfig.java#L20 to be able to automatically switch to `true` when a user configure the host but didn't succeed.

Typical use case would be to not use a central log management tool in local/dev and only in production.